packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.vue

```diff
@@ -49,6 +49,11 @@ export default {
       type: Boolean,
       default: false
     }
+  },
+  computed: {
+    value() {
+      return this.qty >= 1 ? this.qty : 1;
```

Comment:
@justyna-13 probably you can't override qty, you must emit new value to the parent component. Probably you can watch(value) for qty and $emit 1 only when if qty is empty or qty < 1

Reply:
you can find $emit on line 14